cmd/login.go

```diff
@@ -10,9 +12,16 @@ var password string
 
 var loginCmd = &cobra.Command{
 	Use:   "login",
-	Short: "Login to holepunch",
+	Short: "Login to holepunch.",
+	Long: "Login back into holepunch.\n" +
+		"Will prompt you for username and password, or you can provide them as optional arguements.\n" +
+		"You should use `punch setup` instead of `punch login` the first time.",
```

Comment:
"If this is your first time using Holepunch you should use `punch setup` instead of `punch login`.",